doc/releases/changelog-dev.md

```diff
@@ -207,7 +185,10 @@
 
 <h3>Deprecations</h3>
 
-* A deprecation warning has been added to the `qml.finite_diff()` method.
+* The `qml.finite_diff()` function has been deprecated and will be removed
+  in an upcoming release. Instead,
+  `qml.gradients.finite_diff()` can be used to compute purely quantum gradients
+  (that is, gradients of tapes or QNode).
```

Comment:
best to duplicate the warning messages in the changelog :) (@Jaybsoni for reference)

Reply:
Yes! This is much more descriptive and helpful to users than my previous entry. Thanks !